### ISSUE TITLE
entire screen is flashing

### ISSUE THREAD

**Issue by ccd0** [ccd0/4chan-x#921]
Submitter: chris poole
Email: moor@4chan.org
with this extension enabled, 4chan just constantly flashes. all the posts, threads, images, and everything just flash in and out at a steady rhythm

**Comment by JohnFreemason** [comment]
Noticed this issue today as well, literally just happened minutes ago but was working fine earlier in the day.  This only seems to happen on Chrome, regardless of version, and on both the userscript and extension versions.  Tested with all other extensions disabled, resetting all settings, and even with a clean profile.

**Comment by spaghetti2514** [comment]
Stop inserting "blink" to the html tag's classes on Chrome

**Comment by eveo90** [comment]
Confirming on Google Chrome 50.0.2661.102
Experienced problems on user.js v1.11.34.5 build using Tampermonkey, and some other 4chan X having the problem as well including the one on Chrome Web Store

**Comment by spaghetti2514** [comment]
For a quick temporary fix, add ```.blink{animation:none!important;}``` to your custom CSS

**Comment by bakugo** [comment]
https://github.com/Floens/ChanTracking/commit/563e50d26febad9aace37c8e677fc3cc7d8775ef

What are they even doing?

**Comment by eveo90** [comment]
https://boards.4chan.org/b/thread/685475514/go-outside
Might have found the source of the problem

**Comment by ccd0** [comment]
This and #920 #922 #923 #924 #925 #926 #927 #929 #930 #931 #932 #934 #936 should be fixed.
Opened #937 for changing the name of the class.

**Comment by o4zloiroman** [comment]
It's up again.

**Comment by mikeshinobi** [comment]
:stopblinking:

**Comment by dkwan33** [comment]
I am having this blink issue. I believe something is preventing the fix (May 22 Replace blink class #937) from working. Possibly another extension.

I custom injected the ".blink{animation:none!important;}" fix, and it stopped the blinking. I checked styles.css and noticed that this quick fix was removed at some point, in favor of the "replace blink class" fix. Maybe it should be put back in just to be safe?
